framework/core/js/src/common/utils/isDark.ts

```diff
@@ -8,18 +8,24 @@
  * to preserve design consistency.
  */
 export default function isDark(hexcolor: string): boolean {
```

Comment:
```suggestion
export default function isDark(hexcolor: string | null): boolean {
```

could you also type the `textContrastClass` util the same way?